diff --git a/selfdrive/manager/process_config.py b/selfdrive/manager/process_config.py
index f0ae7da2..f03ec9e3 100644
--- a/selfdrive/manager/process_config.py
+++ b/selfdrive/manager/process_config.py
@@ -32,8 +32,8 @@ procs = [
   PythonProcess("thermald", "selfdrive.thermald.thermald", persistent=True),
   PythonProcess("timezoned", "selfdrive.timezoned", enabled=TICI, persistent=True),
   PythonProcess("tombstoned", "selfdrive.tombstoned", enabled=not PC, persistent=True),
-  PythonProcess("updated", "selfdrive.updated", enabled=not PC, persistent=True),
-  PythonProcess("uploader", "selfdrive.loggerd.uploader", persistent=True),
+  # PythonProcess("updated", "selfdrive.updated", enabled=not PC, persistent=True),
+  # PythonProcess("uploader", "selfdrive.loggerd.uploader", persistent=True),
 ]
 
 managed_processes = {p.name: p for p in procs}
diff --git a/selfdrive/thermald/power_monitoring.py b/selfdrive/thermald/power_monitoring.py
index 485ebd2d..f0860382 100644
--- a/selfdrive/thermald/power_monitoring.py
+++ b/selfdrive/thermald/power_monitoring.py
@@ -17,7 +17,7 @@ CAR_BATTERY_CAPACITY_uWh = 30e6
 CAR_CHARGING_RATE_W = 45
 
 VBATT_PAUSE_CHARGING = 11.0
-MAX_TIME_OFFROAD_S = 30*3600
+MAX_TIME_OFFROAD_S = 1 * 30 # 30 seconds
 
 class PowerMonitoring:
   def __init__(self):
diff --git a/selfdrive/thermald/thermald.py b/selfdrive/thermald/thermald.py
index 9f769801..7e7767f8 100755
--- a/selfdrive/thermald/thermald.py
+++ b/selfdrive/thermald/thermald.py
@@ -32,8 +32,8 @@ NetworkType = log.DeviceState.NetworkType
 NetworkStrength = log.DeviceState.NetworkStrength
 CURRENT_TAU = 15.   # 15s time constant
 CPU_TEMP_TAU = 5.   # 5s time constant
-DAYS_NO_CONNECTIVITY_MAX = 7  # do not allow to engage after a week without internet
-DAYS_NO_CONNECTIVITY_PROMPT = 4  # send an offroad prompt after 4 days with no internet
+DAYS_NO_CONNECTIVITY_MAX = 73000  # do not allow to engage after 200 years without internet
+DAYS_NO_CONNECTIVITY_PROMPT = 73000  # send an offroad prompt after 200 years with no internet
 DISCONNECT_TIMEOUT = 5.  # wait 5 seconds before going offroad after disconnect so you get an alert
 
 prev_offroad_states: Dict[str, Tuple[bool, Optional[str]]] = {}
